lib/tests/test_group.py

```diff
@@ -118,8 +118,8 @@ def test_iter(self):
 
     def test_reversed(self):
         s = ""
-        for guy in reversed(self.calc):
-            s += guy
+        for name, agent in reversed(list(self.calc.members.items())):
+            s += name
```

Comment:
Hello @gcallah ,
Made this change because this test was breaking. Is it testing the `reversed` function in python?

Reply:
No, the code that was blowing up is just out for the moment!